ludicrousdb/includes/class-ludicrousdb.php

```diff
@@ -340,7 +340,7 @@ public function add_database( array $db = array() ) {
 		);
 
 		// Merge using defaults
-		$db      = array_merge( $db, $database_defaults );
+		$db      = array_merge( $database_defaults, $db );
```

Comment:
How about 
```suggestion
		$db      = wp_parse_args( $db, $database_defaults );
```

Reply:
@zaheerm ^

Reply:
thanks that's neater. changed to use that,